packages/server/src/ClientConnection.ts

```diff
@@ -84,7 +84,7 @@ export class ClientConnection {
   ) {
     // Make sure to close an idle connection after a while.
     this.closeIdleConnectionTimeout = setTimeout(() => {
-      websocket.close(Unauthorized.code, Unauthorized.reason)
+      websocket.close(IdleTimeout.code, IdleTimeout.reason)
```

Comment:
This timeout is stopped once the connection is fully established, so it's actually more a "timed out waiting for authentication". This can usually happen if the server expects an auth token but the client doesn't send one. 

Totally makes sense to add a separate timeout for this, but I think we should keep the old error message for this.

Reply:
Ah, I see. I may have misunderstood this timeout.